tests/tests/test_management_config.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2021 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import uuid
+import pytest
+
+from common import management_api_with_params
+from internal_api import InternalAPIClient
+
+
+@pytest.fixture
+def device_id():
+    client = InternalAPIClient()
+    device_id = str(uuid.uuid4())
+    new_device = {"device_id": device_id}
+    r = client.provision_device_with_http_info(
+        tenant_id="", new_device=new_device, _preload_content=False
+    )
+    assert r.status == 201
+    yield device_id
+    r = client.decommission_device_with_http_info(
+        tenant_id="", device_id=device_id, _preload_content=False
+    )
+    assert r.status == 204
+
+
+class TestManagementConfig:
+    def test_config_device(self, device_id):
+        user_id = str(uuid.uuid4())
+        client = management_api_with_params(user_id=user_id)
+        #
+        # get the configuration (empty)
+        r = client.get_key_value_pair_store(device_id)
```

Comment:
(Unrelated) Can we change the api spec operation ID to something more succinct (e.g `Get Device Configuration`)? This sounds a bit strange to me.

Reply:
@alfrunes done